packages/sources/eth-balance/src/endpoint/balance.ts

```diff
@@ -111,8 +143,29 @@ export const execute: ExecuteWithConfig<Config> = async (request, _, config) =>
 const getBalance = async (
   address: string,
   targetBlockTag: string | number,
-  config: Config,
+  provider: ethers.providers.Provider,
 ): Promise<AddressWithBalance> => ({
   address,
-  balance: (await config.provider.getBalance(address, targetBlockTag)).toString(),
+  balance: (await provider.getBalance(address, targetBlockTag)).toString(),
 })
+
+// const getBalanceMultiChain = async (
+//   address: string,
+//   targetBlockTag: string | number,
+//   config: Config,
+//   chainId: string,
+//   jobRunID: string | undefined,
+// ): Promise<AddressWithBalance> => {
+//   const provider = config.chainIdToProviderMap.get(chainId)
+//   if (!provider) {
+//     throw new AdapterInputError({
+//       jobRunID,
+//       message: `Missing provider mapping for chainId ${chainId}.`,
+//       statusCode: 400,
+//     })
+//   }
+//   return {
+//     address,
+//     balance: (await provider.getBalance(address, targetBlockTag)).toString(),
+//   }
+// }
```

Comment:
Still needed? If not, can we just delete?

Reply:
not needed, thx. 2am brain miss